docusaurus.config.js

```diff
@@ -38,6 +38,17 @@ const config = {
 
   plugins: ["docusaurus-plugin-sass"],
 
+  themes: [
+    [
+      require.resolve("@easyops-cn/docusaurus-search-local"),
+      /** @type {import("@easyops-cn/docusaurus-search-local").PluginOptions} */
+      {
+        hashed: true,
+        indexPages: true,
```

Comment:
I have added this so the integrations page, which isn't part of the docs, is also indexed.